src/prefect/utilities/importtools.py

```diff
@@ -185,12 +197,15 @@ def import_object(import_path: str):
     - module.object
     - module:object
     - /path/to/script.py:object
+    - /path/to/object.pickle
```

Comment:
What are your thoughts on `pickle:///path/to/object` instead?

Reply:
That seems intuitive, however the `file://`-format is typically used to specify format (`s3:///bucketName/path/to/blob` for S3 for example).

What do you think?

Reply:
Hm, I'm thinking of how fsspec chains urls to indicate compression (https://filesystem-spec.readthedocs.io/en/latest/features.html#url-chaining)

They'd do `zip::s3://....` to indicate reads/writes should work with compressed content. Similarly, a `pickle::file:///...` makes sense to me. I'm hesitant to rely to the file extension, but not entirely opposed. I'll see if other people on our team have thoughts.

Reply:
the fsspec-scheme seems like a good idea. Relying on the file-extensions is indeed not ideal, especially as Prefect 1 used the extension `.flow` for pickled flows.